examples/cicd/golang/manifests.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: cyclone.io/v1alpha1
+kind: Resource
+metadata:
+  name: example-codes
+spec:
+  type: Git
+  parameters:
+  - name: GIT_URL
+    value: https://github.com/cd1989/code-zoo.git
+  - name: GIT_REVISION
+    value: master
+
+---
+
+apiVersion: cyclone.io/v1alpha1
+kind: Resource
+metadata:
+  name: example-image
+spec:
+  type: Image
+  parameters:
+  - name: IMAGE
+    value: test.caicloudprivatetest.com/release/golang-example:v0.0.1
+  - name: IMAGE_FILE
+    value: image.tar
+
+---
+
+apiVersion: cyclone.io/v1alpha1
+kind: Stage
+metadata:
+  name: build
+spec:
+  pod:
+    inputs:
+      resources:
+      - name: example-codes
+        path: /go/src/examples
+    spec:
+      containers:
+      - name: main
+        image: golang:1.10-alpine3.8
+        workingDir: /go/src/examples/golang
+        command:
+        - /bin/sh
+        - -c
+        - "go build -o ./bin/app . && ./bin/app"
+
+---
+
+apiVersion: cyclone.io/v1alpha1
+kind: Stage
+metadata:
+  name: image-build
+spec:
+  pod:
+    inputs:
+      resources:
+      - name: example-codes
+        path: /go/src/examples
+    outputs:
+      resources:
+      - name: example-image
+        path: /go/src/examples/golang/image.tar
```

Comment:
Will the image this stage built be pushed to registry?

Reply:
Yes